src/fragments/lib/auth/android/social_signin_web_ui/10_cli_setup.mdx

```diff
@@ -1,4 +1,10 @@
-In terminal, navigate to your project, run `amplify add auth`, and choose the following options (the last steps are specific to Facebook here but are similar for other providers):
+Once you have the social provider configured, run the following in your project’s root folder
+
+```bash
```

Comment:
does it look different `bash` than `terminal`

Reply:
We were [previously using](https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/auth/js/social.mdx?plain=1#L15) `bash` and `terminal` on the Social sign-in (OAuth) Js page, so maintained the same.

Currently looks (bash, console)
<img width="600" alt="Screen Shot 2022-08-08 at 7 53 42 PM" src="https://user-images.githubusercontent.com/22762119/183553439-bd5534a5-063b-497b-ad54-c3d0ecf4da48.png">

 If both are console
<img width="600" alt="Screen Shot 2022-08-08 at 7 55 39 PM" src="https://user-images.githubusercontent.com/22762119/183553813-7cc338f1-a9f9-432b-8f17-7d4a972531ea.png">
 
 If both are bash
<img width="600" alt="Screen Shot 2022-08-08 at 7 58 30 PM" src="https://user-images.githubusercontent.com/22762119/183553947-24e57753-f581-4285-94bc-aa8bcf582d3e.png">

Reply:
I think we should just use what it previously was - since it seems like the whole docs site right now uses terminal.